assets/i18n/tr.json

```diff
@@ -0,0 +1,299 @@
+﻿{
```

Comment:
Oh wait, sorry -- undo LGTM.

The Travis checks appear to be [failing](
https://travis-ci.org/oppia/oppia/jobs/179422649#L2141). It seems that, somehow, you have a bunch of invisible characters before the "{": see the log in the previous link for details. This appears to be an issue with some editors on Windows: https://github.com/bkeepers/dotenv/issues/224.

Can you make sure that these are removed? I'm not sure what editor program you are using, but it might be adding junk characters to the file.

To make sure it works, you should run: `bash scripts/run_backend_tests --test_target=core.controllers.base_test.I18nDictsTest` and only submit after that test passes.

Also, did you set up and run a test instance locally? That could also help to guard against errors like this.

Thanks!